src/roman_datamodels/testing/factories.py

```diff
@@ -457,6 +569,30 @@ def create_guidestar(**kwargs):
     return stnode.Guidestar(raw)
 
 
+def _create_basic_meta():
+    """
+    Create the metadata from the basic-1.0.0 schema, which is shared
+    between references and datasets.
+
+    Returns
+    -------
+    dict
+    """
+
+    return {
+        "calibration_software_version": _random_string("Version ", 120),
+        "crds_context_used": "roman_{:04d}.pmap".format(_random_positive_int(9999)),
+        "crds_software_version": _random_software_version(),
+        "filename": _random_string("Filename ", 120),
+        "file_date": _random_astropy_time(),
```

Comment:
Not an issue now I suspect, but aren't random dates problematic around CRDS? (If this ever tests against CRDS, that is..)

Reply:
Yeah, they would be.  I expect that the random values will be refined over time as they prove to be problematic (ideally with associated changes to the schemas).